src/System.Numerics.Vectors/src/System/Numerics/Register.tt

```diff
@@ -5,7 +5,7 @@
 <#@ import namespace="System" #>
 <#@ import namespace="System.Linq" #>
 <#@ import namespace="System.Runtime.InteropServices" #>
-<#@ import namespace="System.Diagnostics.Contracts" #>
```

Comment:
Are there no .cs files that need to be regenerated based on these template changes?  And if not, does that mean Register.tt is dead code?